### ISSUE TITLE
float and double properties in complex types not showing up in swagger Model Schema

### ISSUE THREAD

**Issue by makimat** [domaindrivendev/Swashbuckle#316]
Can someone else reproduce the same problem?

I have specified a model like this:

```csharp
namespace MyNamespace
{
    public class MyComplexType
    {
        public string Parameter1 { get; set; }
        public double Parameter2 { get; set; }
    }
}
```

And use it as an input to a POST request:

```csharp
namespace MyNamespace
{
    [RoutePrefix("api")]
    public class MyController : ApiController
    {
        [Route("request1")]
        [HttpPost]
        public HttpResponseMessage Request1(MyComplexType data)
		{
			// ...
		}
	}
}
```

In the generated swagger json Parameter2 (double) shows up as

```json
... "Parameter1":{"type":"string"},"Parameter2":{"format":"double","type":"number"} ...
```

But in the rendered documentation for /api/request1, Parameter2 does not show up in Model Schema:

```json
{
  "Parameter1": "string"
}
```

If I toggle open Model instead of Model Schema, then both parameters show up:

```
MyNamespace.MyComplexType {
  Parameter1 (string, optional),
  Parameter2 (number, optional)
}
```

How come?

**Comment by domaindrivendev** [comment]
This is a dup of #282

Fixed in the latest version (5.1.5)

**Comment by makimat** [comment]
Thanks a lot, will update.
